src/main/java/model/Fruit.java

```diff
@@ -0,0 +1,31 @@
+package model;
+
+import strategy.OperationHandler;
+
+public class Fruit {
+    private final OperationHandler operation;
```

Comment:
```suggestion
public class FruitTransportation {
    private final Operation operation;
```
хендлер не може бути полем можелі ( це порушення принципів SOLID)  стратегія реалізується на рівні сервісів